### PR TITLE
Allow to pass body to apiRequest

### PR DESCRIPTION
**Pull request by viktor-yakubiv**
Slightly improves code structure.

Adds description comments.

Comes from #148

### CODE REVIEW

File: api/index.js [hunk 1]
```diff
@@ -7,25 +7,39 @@ const apiRequest = (
...
+      Accept: 'application/json',
+    },
+  }
+
+  // Process body
+  if (method.toUpperCase() !== 'GET') {
+    if (typeof data == 'object') {
+      requestOptions.body = JSON.stringify(data)
+      requestOptions.headers['Content-Type'] = 'application/json'
+    } else requestOptions.body = data
```
Comment (Joozty) [inline api/index.js]: Don't you want to pass `text/plain` for this case?

File: api/index.js [hunk 2]
```diff
@@ -7,25 +7,39 @@ const apiRequest = (
...
-    signal,
-  })
+    headers: {
+      Accept: 'application/json',
+    },
+  }
+
+  // Process body
+  if (method.toUpperCase() !== 'GET') {
+    if (typeof data == 'object') {
```
Comment (Joozty) [inline api/index.js]:
```suggestion
    if (typeof data === 'object') {
```

Comment (viktor-yakubiv) [reply api/index.js]: We use `eqeqeq: smart` for a reason. `typeof` always returns a string, so if you compare it to another string, there is no need to write another character.

File: api/index.js (hunk 1 above)
Comment (viktor-yakubiv) [reply api/index.js]: I leave the responsibility to set correct content type up to the user.

However, I've just checked, `text/plain` is default and there is no need in setting it explicitly.

**Pull request merged**